src/renderer/experimental.js

```diff
@@ -51,6 +51,13 @@ export const experimentalFeatures: Feature[] = [
     title: "Experimental device flow",
     description: "Automatically asks an Open App device permission in device flows",
   },
+  {
+    type: "toggle",
+    name: "EXPERIMENTAL_EXPLORERS",
+    title: "Experimental Explorers API",
+    description:
+      "Try an incoming version of Ledger explorers. Changing this value can affect the Account balance, synchronisation and send feature.",
```

Comment:
"Try an upcoming version of Ledger's blockchain explorers. Changing this setting may affect the account balance and synchronization as well as the send feature.",

Reply:
thanks! done